docker-compose.yml

```diff
@@ -36,7 +37,7 @@ services:
         target: /app
   applicants:
     build: ./bops-applicants
-    entrypoint: ./bops-applicants/docker-entrypoint.sh
```

Comment:
This should stay as by changing it to `./docker-entrypoint.sh` it will use the `./docker-entrypoint.sh` from bops, not from `bops-applicants`

Reply:
Does it work for you doing this though?

I'm getting an error

Reply:
I pasted the error message here: https://github.com/unboxed/bops/pull/555/commits/0d5f9d770593097839682290981e3effbdc1246f